hydra/helpers/client.py

```diff
@@ -186,14 +213,52 @@ def configure(self, name, destination, **kwargs):
             'chaindata/config/genesis.json'
         )
 
+        # LOOM.YAML
+        self._copy_yaml(
+            f'{self.app.config["hydra"]["channel_url"]}/networks/{name}/loom.yaml',
+            'loom.yaml'
+        )
+        if kwargs['oracle'] if 'oracle' in kwargs else False:
```

Comment:
For the sake of keeping this growing method easier to read, can we extract this Oracle loom.yaml setup to its own method `_setup_oracle_loom_yaml`?